### DISCUSSION TITLE
Collection of requirements, ideas, and features for Harbor 2.11

### DISCUSSION THREAD

**Discussion by Vad1mo** [goharbor/harbor#19791] [Ideas]
We invite the community to comment and vote on the next feature for Harbor 2.11

###  Focus Areas

here are some focus areas the Harbor team identified for 2.11
- focus on SBOM - https://github.com/goharbor/harbor/issues/19130
- 2.11 candidate list - https://github.com/goharbor/harbor/labels/candidate%2F2.11.0

### Timeline

- Feature Freeze: 22 Jan 2024
- Target FC: 18 Mar 2024
- Target RC1: 01 Apr 2024
- Target RC2: 08 Apr 2024
- Target GA: 15 Apr 2024

Here are some references to previous discussions and currently planned issues:
- [Candidate 2.11.0 Issues](https://github.com/goharbor/harbor/issues?q=is%3Aissue+is%3Aopen+label%3Acandidate%2F2.10.0)
- https://github.com/goharbor/harbor/discussions/18396
- https://github.com/goharbor/harbor/discussions/19238
- [Issues flagged as kind of Requirements](https://github.com/goharbor/harbor/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Frequirement)

Please also use the up arrow and the thumbs  👍 👎 to help us rank your requests.

**Comment by ivansmm** [comment]
Hallo, is it possible to fix in 2.11 registry search problem, described in "problem with registry search #19724" issue?

**Comment by thcdrt** [comment]
Hello,

I'm still interested (and I think @lengrongfu too) into a system global quota feature which would block all the artifacts push when the configured quota is reached.
See https://github.com/goharbor/harbor/issues/18011 or https://github.com/goharbor/harbor/issues/16671.

Thomas

**Comment by Vivekgaddigi** [comment]
how can i contribute to this most of these are already assigned?

**Comment by rgarcia89** [comment]
I would like to see the cleanup process to delete empty repositories
